docs/release-notes/release-notes.md

```diff
@@ -2,6 +2,26 @@
 
 This guide provides notes for major version releases. These notes may be helpful for users when upgrading from previous major versions.
 
+## v4.0.0
+
+### Node Operators (v4.0.0)
+
+#### Multiplexer
+
+Celestia-app v4.0.0 introduces support for a [multiplexer](https://github.com/celestiaorg/celestia-app/tree/e5d5ac6732c55150ea3573e17bec162fe836e0c6/multiplexer) that makes it easier for node operators to run a consensus node that can sync from genesis. The multiplexer contains an embedded celestia-app v3.x.x binary that will be used to sync the node from genesis. After the chain advances to app version v4, the multiplexer will stop routing requests to the embedded celestia-app v3.x.x binary and will instead begin routing requests to the celestia-app v4.x.x binary. Binaries that are installed from source (e.g. via `make install`) or via the pre-built binaries attached to the release will include the multiplexer. To install Celestia without the multiplexer, you can use the `make install-standalone` target. Note that the standalone binary will only be able to run on networks that have already upgraded to app version v4.
+
+#### `rpc.grpc_laddr`
+
+The `rpc.grpc_laddr` config option is now required when running the celestia-app binary with the multiplexer. This option can be set via CLI flag `--rpc.grpc_laddr tcp://0.0.0.0:9098` or in the `config.toml`:
```

Comment:
Is it actually needed to make this listen publicly? Seems scary if so. Can we discuss in an issue?